libs/wallet/src/use-injected-connector.ts

```diff
@@ -11,28 +17,42 @@ export enum Status {
   AcknowledgeNeeded = 'AcknowledgeNeeded',
 }
 
+export const ERR_VEGA_UNDEFINED = new Error('window.vega not found');
```

Comment:
suggestion (non blocking): I think its nicer for these to be objects, it makes finding the right error via auto-import/auto completion a little clearer

```
export const InjectedConnectorErrors = {
  VEGA_UNDEFINED: new Error('window.vega not found'),
  ....
}
```